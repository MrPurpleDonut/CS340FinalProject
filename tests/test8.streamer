until text filecontents << input "Enter file: ";
{
  print "Invalid file";
}

print filecontents;
