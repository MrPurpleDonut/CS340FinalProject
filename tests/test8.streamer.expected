Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Until Statement
            Declaration statement: filecontents
                Type: text
                File Read into variable: filecontents
                    Input expression
                        Quoted string: "Enter file: "
            Statement List
                Print Statement
                    Quoted string: "Invalid file"
        Print Statement
            Identifier: filecontents
